pdf.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+# -*- coding: utf8 -*-
+from fpdf import FPDF
+
+def fix_title(s):
+    return s.replace('_', ' ').capitalize()
+
+def generate_pdf(personality, id):
+    pdf = FPDF(format='A4')
+    pdf.add_page()
+    pdf.add_font('PTSans', '', 'static/fonts/PTSans-Regular.ttf', uni=True)
+    pdf.add_font('PTSans', 'B', 'static/fonts/PTSans-Bold.ttf', uni=True)
+
+    pdf.image('static/img/pdf_header.png', 0, 0, 210)
+    pdf.set_font('PTSans', 'B', 12)
+    pdf.cell(60, 25, 'https://trollfactory.tk/' + str(id), 0, 1, '')
+    
+    for i in range(10):
```

Comment:
is the pdf height always the same? what happens if we add more props?

Reply:
the pattern is only visible on the first page, which is always the same height